web/moduly/finance.php

```diff
@@ -73,16 +73,11 @@
         <?= $veci ?>
     </div>
 
-    <?php if ($u && $u->jeOrganizator()) { ?>
+    <?php if ($u?->jeOrganizator() || $u?->jeVypravec()) { ?>
```

Comment:
Proč ne prostě tady nedat
```suggestion
    <?php if ($slevyA || $slevyV) { ?>
```
?